### DISCUSSION TITLE
N64 Dump Issue

### DISCUSSION THREAD

**Discussion by majorpbx** [sanni/cartreader#200] [Need help]
Not sure if this is a true issue but figured I had an odd experience so I'll share regardless. I was dumping my copy of "Ganbare Goemon - Neo Momoyama Bakufu no Odori (Japan)" and no matter how much cleaning I did to the cartridge pins it will not recognize the cartridge. The Cart Reader only displays "ID:  NG5J" and says "Cartridge unknown". I even tested all the cartridge fingers and they are all in excellent condition. The odd part about it is if you select "16MB" (which is the size of the rom) it reads it perfectly. There is no save chip on this cart so that's not a problem. If this is a known issue and I missed it please let me know.

**Comment by Androxilogin** [comment]
I'm pretty sure this means it has yet to have been added to the n64.txt is
all. I get the same with 40 Winks but the rom works correctly.

**Comment by sanni** [comment]
Added it to the n64.txt database.

**Comment by majorpbx** [comment]
Is this project able to use No-Intro DAT's? If not can I suggest adding that feature? It would be easier if you didn't have to update your own database and just say download it and add it to the cart reader.

Just a thought.
